pkg/jobs/job_scheduler.go

```diff
@@ -351,6 +351,16 @@ func (s *jobScheduler) executeSchedules(
 	return err
 }
 
+// An internal, safety valve setting to revert scheduler execution to distributed mode.
+// This setting should be removed once scheduled job system no longer locks tables for excessive
+// periods of time.
+var schedulerRunsOnSingleNode = settings.RegisterBoolSetting(
+	settings.TenantReadOnly,
+	"jobs.scheduler.single_node_scheduler.enabled",
+	"execute scheduler on a single node in a cluster",
+	true,
```

Comment:
I thought we were making this default false

Reply:
Should it default true for v22.1 and false for the v21.2 backport?

Reply:
Ooops... Sorry, fixed.